src/python/aqueduct_executor/operators/connectors/data/snowflake.py

```diff
@@ -20,3 +59,19 @@ def _create_engine(config: config.SnowflakeConfig) -> engine.Engine:
         warehouse=config.warehouse,
     )
     return create_engine(url)
+
+
+def _map_to_snowflake_mode(params: load.RelationalParams) -> str:
+    """
+    Map Aqueduct update modes to Snowflake update modes found here:
+    https://docs.snowflake.com/ko/developer-guide/snowpark/reference/python/api/snowflake.snowpark.DataFrameWriter.save_as_table.html
+    """
+    snowflake_update_mode = "ignore"
+    if params.update_mode.value == "append":
```

Comment:
replace these with the enums defined [here](https://github.com/aqueducthq/aqueduct/blob/main/src/python/aqueduct_executor/operators/connectors/data/common.py#L23)